l10n_account_translate/account_fix.py

```diff
@@ -0,0 +1,83 @@
+# -*- encoding: utf-8 -*-
+# noqa: account_account, name_search is a backport from Odoo.
+#       OCA has no control over style here.
+# flake8: noqa
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#
+#    Copyright (c) 2014 Noviat nv/sa (www.noviat.com). All rights reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models
+from openerp.osv import expression
+
+
+class account_account(models.Model):
+    _inherit = 'account.account'
+
+    def name_search(self, cr, user, name,args=None, operator='ilike', context=None, limit=100):
+        """
+        temporary fix:
+        method copied from standard account module and modified to add multi-language support on name field
+        cf. https://github.com/odoo/odoo/pull/4511
+        """
+        if context is None:
```

Comment:
Since this part of the module should be removed as soon as dissapear as soon as PR https://github.com/odoo/odoo/pull/4511 is merged by odoo (not only modules using this 'l10n_account_translate' module but also localisation modules based upon the official addons 'l10n_multilang' such as the swiss localisation are affected by this, hence it's something that should be merged by Odoo).